connctx/connctx.go

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 // Package connctx wraps net.Conn using context.Context.
+// Deprecated: use netctx instead.
 package connctx
```

Comment:
There are packages that currently depend on this, so we deprecate while introducing the same functionality in `netctx`.